snuba/cli/config.py

```diff
@@ -55,18 +57,32 @@ def get(*, key: str, format: str) -> None:
 @config.command()
 @click.argument("key")
 @click.argument("value")
-def set(*, key: str, value: str) -> None:
+@click.option(
+    "--force", is_flag=True, default=False, help="Override type checking of values"
```

Comment:
I would call this `force-type` since it is meant for a specific case that you want to force.

Reply:
Updated